com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/CustomPass/CustomPassBookmarkEventContext.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace UnityEngine.Rendering.HighDefinition
```

Comment:
Will add proper docs here, if you guys agree on this callback arguements / naming